DESCRIPTION

```diff
@@ -21,6 +21,6 @@ Encoding: UTF-8
 LazyData: true
 RoxygenNote: 7.2.3
 Depends: R (>= 3.5.0)
-Imports: data.table, methods, stats, robustbase, lubridate, terra, geodata
+Imports: data.table, methods, stats, robustbase, lubridate, terra, geodata, testthat
```

Comment:
are you sure we need this as a permanent dependency? Imo this is only needed for testing, not at runtime?